app/lib/dns.server.ts

```diff
@@ -0,0 +1,200 @@
+import {
+  Route53Client,
+  CreateHostedZoneCommand,
+  ChangeResourceRecordSetsCommand,
+  GetChangeCommand,
+} from "@aws-sdk/client-route-53";
+import {
```

Comment:
Use `import type` and also move this to its own separate line, below the other regular imports.